File: src/main/java/com/okanmenevseoglu/airportinfomanager/dto/response/AirportInformationReportResponse.java
package com.okanmenevseoglu.airportinfomanager.dto.response;

import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class AirportInformationReportResponse {

    private List<CountryWithAirportCountResponse> countriesWithHighestAirportCount;

    private List<CountryWithAirportCountResponse> countriesWithLowestAirportCount;

    private List<TypeOfRunwayByCountryResponse> typeOfRunwaysByCountry;

    private List<RunwayIdentificationWithCountResponse> runwayIdentificationsWithCount;
}

File: airport-info-client/src/app/shared/response/airport-information-report-response.ts
import {CountryWithAirportCountResponse} from './country-with-airport-count-response';
import {TypeOfRunwayByCountryResponse} from './type-of-runway-by-country-response';
import {RunwayIdentificationWithCountResponse} from './runway-identification-with-count-response';

export interface AirportInformationReportResponse {

  countriesWithHighestAirportCount: CountryWithAirportCountResponse[];

  countriesWithLowestAirportCount: CountryWithAirportCountResponse[];

  typeOfRunwaysByCountry: TypeOfRunwayByCountryResponse[];

  runwayIdentificationsWithCount: RunwayIdentificationWithCountResponse[];
}
